tests/orquestra/integrations/cirq/conversions/circuit_conversions_test.py

```diff
@@ -260,8 +260,9 @@ def test_binding_and_exporting_parametrized_circuit_results_in_equivalent_circui
         bound = orquestra_circuit.bind(EXAMPLE_PARAM_VALUES)
         bound_converted = export_to_cirq(bound)
         ref_bound = cirq.resolve_parameters(
-            cirq_circuit, {**EXAMPLE_PARAM_VALUES, sympy.pi: 3.14}
+            cirq_circuit, {**EXAMPLE_PARAM_VALUES}  # , sympy.pi: 3.14}
```

Comment:
I had to remove `sympy.pi: 3.14` since cirq is not accepting. Not sure if this was kept there for any other purpose.

Reply:
👌 
Please remove the commented code.